### ISSUE TITLE
npm audit fail on last version (5.11.0) due to cookie(0.5.0) vulnerability

### ISSUE THREAD

**Issue by jpikerun** [openapistack/openapi-backend#750]
The following audit report is generated with the latest openapi-backend version.

```
# npm audit report

cookie  <0.7.0
cookie accepts cookie name, path, and domain with out of bounds characters - https://github.com/advisories/GHSA-pxg6-pf52-xh8x

node_modules/cookie
  openapi-backend  >=1.3.6
  Depends on vulnerable versions of cookie
  node_modules/openapi-backend
```

**Comment by anttiviljami** [comment]
Merged PR #751 to fix the issue 👍

**Comment by anttiviljami** [comment]
Published the fix as part of `openapi-backend@5.11.1`

**Comment by jpikerun** [comment]
Thank you
